troposphere/settings/local.py.j2

```diff
@@ -17,12 +17,15 @@ API_V2_ROOT = API_SERVER + "/api/v2"
 
 SITE_TITLE= "{{ SITE_TITLE}}"
 SITE_FOOTER= "{{ SITE_FOOTER }}"
+
 {% if THEME_NAME is defined and not THEME_NAME == "default" %}
 THEME_NAME="{{ THEME_NAME }}"
 {% else %}
-THEME_NAME="troposphere_theme"
+THEME_NAME= "troposphere_theme"
```

Comment:
So my thoughts here are that the "troposphere_theme" that we are referring to should *really* be the 'neutral theme'.. The values/icons/etc. that one gets when *no* theme has been applied. In this case we are applying a cyverse theme and thus the name change here..

My :two: cents